Frameworks/UIKit.Xaml/ScrollView.xaml.cpp

```diff
@@ -0,0 +1,89 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "pch.h"
+#include "ScrollView.xaml.h"
+
+using namespace Microsoft::WRL;
+using namespace Platform;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+
+namespace UIKit {
+namespace Xaml {
+
+// clang-format does not like C++/CX
+// clang-format off
+
+ScrollView::ScrollView()
+{
+    InitializeComponent();
+
+    SetValue(UIKit::Xaml::Private::CoreAnimation::Layer::LayerContentProperty, LayerContentImage);
+    SetValue(UIKit::Xaml::Private::CoreAnimation::Layer::SublayerCanvasProperty, ContentCanvas);
```

Comment:
I spent quite some time on this and i could not get it working if i don't do this. It reports could not find this property? is there any trick here? 

---
In reply to: [106705678](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106705678) [](ancestors = 106705678)